util/app-config/src/cli.rs

```diff
@@ -238,6 +239,7 @@ fn init() -> App<'static, 'static> {
             Arg::with_name(ARG_BA_CODE_HASH)
                 .long(ARG_BA_CODE_HASH)
                 .value_name("code_hash")
+                .validator(|code_hash| is_hex(code_hash.as_str()))
```

Comment:
change the `is_hex` fn parameter type to `String` according to validator closure type:

```suggestion
                .validator(is_hex)
```